Common/src/grid_movement_structure.cpp

```diff
@@ -7871,11 +7940,24 @@ CFreeFormDefBox::CFreeFormDefBox(unsigned short val_lDegree, unsigned short val_
 			}
 		}
 	}
-	 
+
+  BlendingFunction = new CFreeFormBlending*[nDim];
```

Comment:
Are there any problems here if nDim = 2, since length(BlendingFunction) is assumed 3 just below?